### DISCUSSION TITLE
renew my github student pack

### DISCUSSION THREAD

**Discussion by devazizulhaque** [community/community#147481] [GitHub Education]
### Hi everyone,

hi recently i can not use github education still now i am student

**Comment by milekv** [comment]
If you are still a student but have lost access to [GitHub Education](https://github.com/education), you most likely need to renew your status verification. Visit GitHub Education and see if your account needs to be updated. Prepare documents such as your student ID or university certificate and resend them. Also make sure your university email is correctly linked to your account. If you have any problems, please contact GitHub support via support. 😊
